test/interop/C/llvm/makefiles/requireStmt.future

```diff
@@ -1 +0,0 @@
-bug: support --library-makefile with LLVM
```

Comment:
I believe there is one more future that is missing.  It lives in the test/interop/C/llvm/noLibFlag directory (which checks the behavior of the `--library-*` flags when `--library` itself is omitted.  Basically, the intention is that these flags will turn on `--library` compilation if it is not explicitly turned on, since the flag names are long and something a user would only want with `--library` compilation).

Note that when checking the C backend as mentioned in my earlier comment, you will also want to run the C equivalent (so test/interop/C/noLibFlag)